elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
   title--m-sm--FontSize:    #{pfe-var(FontSize--sm ) },   // 14px
   title--m-xs--FontSize:    #{pfe-var(FontSize--xs ) },   // 12px
 );
+
+$pf-typography-sets: (
+  title: (
+    "6xl": (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
```

Comment:
I'm putting notes here so I can remember things I want to ask you, but we can chat about it verbally too :) 
I'm wondering why map-get here vs the pfe-var function? Is it because of order of operations, or that its one map referencing another and pfe-var leans on maps, so it wont work?

Reply:
👍 I didn't want to delete the new maps you built so I referenced them instead but I love the idea of just 1 map that is the source of truth

Reply:
If we moved all the variables into one map, I think it makes it harder to use `map-get` or `pfe-var` to use some values to populate others. Which is how I ended up with multiple maps.

My question above though, was more around use of the `map-get` function vs the `pfe-var` function, if there was a benefit of one over the other?

Reply:
Have you checked out `map-deep-get`?  I don't think map-get is difficult if you have a map-deep-get set up. `map-deep-get($pf-typography-sets, title, 6xl, FontSize)` -> it almost reads like a sentence.

Reply:
You can use pfe-var or map-get.  Since they're in the same file, map-get just seems more intuitive to me and requires less processing.  pfe-var assumes you don't know what map the data you want is in and checks several for you.

Reply:
Cool, that makes sense :)